urllib3/response.py

```diff
@@ -126,8 +126,8 @@ def __init__(self, body='', headers=None, status=0, version=0, reason=None,
         # Are we using the chunked-style of transfer encoding?
         self.chunked = False
         self.chunk_left = None
-        tr_enc = self.headers.get('transfer-encoding', '')
-        if tr_enc.lower() == "chunked":
+        tr_enc = self.headers.get('transfer-encoding', '').lower()
+        if "chunked" in tr_enc.split(", "):
```

Comment:
It's probably better to split on comma, and then strip each element. In practice the space-separator is a bit hit and miss.